apple/MarkdownTextFieldObserver.h

```diff
@@ -0,0 +1,19 @@
+#import <UIKit/UIKit.h>
+#import <React/RCTUITextField.h>
+#import <RNLiveMarkdown/RCTMarkdownUtils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MarkdownTextFieldObserver : NSObject
+
+@property (nonatomic, nullable, strong) RCTMarkdownUtils *markdownUtils;
+
+@property (nonatomic, nullable, strong) RCTUITextField *textField;
```

Comment:
And those two look like they could be set in the constructor (asserts can be moved there then).

Reply:
Added nice constructors in 40abeba